content/millennium/r4/financial/charge-item.md

```diff
@@ -44,6 +44,7 @@ The following fields are returned if valued:
 * [Offset By]
 * [Performing Location]
 * [Procedure]
+* [Replacing]
```

Comment:
I think we may need Replacing listing in the Extensions section of "The following fields are returned if valued:" in the Overview

Reply:
Added [here](https://github.com/cerner/fhir.cerner.com/pull/626/commits/a8d4a0280e037886cfdcd8149eb3eb1e4abceb6a#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR32).
Screenshot updated above.

Reply:
Thanks